README.md

```diff
@@ -90,20 +96,23 @@ As copyright owner, ZeroC can license Ice under different license terms, and off
 - Commercial or closed-source licenses
 
 If you license Ice under GPL v2, there is no license fee or signed license agreement: you just need to comply with the
-GPL v2 terms and conditions. See [ICE_LICENSE](ICE_LICENSE) and [LICENSE](LICENSE) for further information.
+GPL v2 terms and conditions. ZeroC also grants a few [exceptions](ICE_LICENSE) to the GPLv2 terms and conditions.
```

Comment:
This is inconsistent with how we write `GPL v2` everywhere else in this section.
I prefer `GPLv2`, but if we're going to add it here, we should change the other uses to still be consistent.
Or change this one to match the othes.

Reply:
Fixed. It's now spelled "GPLv2".